### DISCUSSION TITLE
More morphs

### DISCUSSION THREAD

**Discussion by srnyx** [commandgeek/geeksmp#49] [Ideas]
### What and How?

More morphs for undead. **Ideas welcome in comments!**

### Why?

Would be nice to have more options for the undead.

### Are there alternatives?

Improving the current ones (https://github.com/commandgeek/geeksmp/discussions/64).

**Comment by DWAA1660** [comment]
if you do an elytraa thing for a phantom morph youll have to rewrite my code about undeads not being able to have elytra

**Comment by techred32** [comment]
### Spider
Spider would be a great idea. Maybe the undead could climb up like a five block wall. This would be super useful when getting to players with roles.

**Reply by srnyx** [reply]
Yeah, was thinking the same thing.
Spiders could have bows that summon a web upon landing.
They could climb walls and maybe even crawl (without requiring blocks above them) whenever they want by using a command or something, not sure how it would be activites.

**Comment by MillkSheik** [comment]
### Creeper
Creeper would be a cool idea, right click with a gunpowder or something and u would explode (with no block griefing) and die, but if u killed someone u would respawn as a revived, idk just ideas

**Comment by srnyx** [comment]
### Phantom
Not sure what the abilities would be. Maybe a permanent elytra (that you can't remove)?

**Comment by srnyx** [comment]
### Witch
One of the slots would get a random potion every time one is thrown. This would be a locked slot in the hotbar (slot 9 [very end] would be best). I'm pretty sure you did this with the Epic YouTubers Collab datapack for one of the YouTubers.
You could even have more than one potion "category." A "good" category, and a "bad" category. The bad category would consist of harmful (poison, instant damage, etc...) splash potions. The good category would consist of helpful (speed, jump boost, fire resistance, etc...) bottled potions.
Maybe they could also spawn with a horse or something named `Broom` that's fast lmao

**Comment by FlyingDrag0n** [comment]
Here are some interesting mobs that would be cool the morph into:

### Silverfish

1. They can go under half a block (if not possible maybe like speed 5 permanent)
2. Does only about 0.5 dmg
3. Can walk on water

### Fox

1. Jumps high
2. Is quick (Speed 3 or so)
3. When near fire (campfires, fire, etc...) it gets regeneration

### Bee

1. It can fly every 30s
2. It drops out honeycombs when they're near a flower(cooldown of 10m or so, kinda useless but funny)
3. Poison players/mobs when hitting them
4. Dies of water

### Illuisoner

1. Sends out clones of the player (5x)
2. Gets a free bow + infinite arrows (like skeleton)
3. Gives out blindness to players when they're near the Illusioner of 50 blocks.
4. Permanent slowness 1


If you got better idea's please comment down below me. I like to read what you think of it or if you wanna improve/disprove it.

**Reply by srnyx** [reply]
- **Bee**
  - **2:** Could probably be abused easily to dupe (even though it may be a pretty useless item it's still duping nonetheless)
  - **3:** Would probably be good if it had like a 1-minute or 5-minute cooldown. And since bees die after they sting maybe the bee could take half the damage it dealt.
- **Illusioner**
  - **3:** 50 blocks is actually quite a lot if you measure in-game. Would probably be better if it was somewhere between 10 and 20.

**Reply by FlyingDrag0n** [reply]
> * **Bee**
>   * **3:** Would probably be good if it had like a 1-minute or 5-minute cooldown. And since bees die after they sting maybe the bee could take half the damage it dealt.

The timer cooldown could be a good idea. Probably after like 3-5m. And they take like 1 heart or so.

> * **Illusioner**
>   * **3:** 50 blocks is actually quite a lot if you measure in-game. Would probably be better if it was somewhere between 10 and 20.

Yeah.. It might need to be shorten down. But I think its an awesome new thing where players will be blind and confused. So maybe 15 could work or 20

**Comment by TigerGameplay** [comment]
### Wandering Trader
We should have a **trader** who gets speed and protection but weakness so it will be bad in combat but good abilities to go around and explore. Might also be usefull to get your items back when died.
